src/main/java/appeng/util/Platform.java

```diff
@@ -160,9 +160,9 @@ public class Platform {
             UUID.fromString("839eb18c-50bc-400c-8291-9383f09763e7"),
             "[AE2Player]");
     private static final String[] BYTE_UNIT = { "B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB", "BB" };
-    private static final String[] NUM_UNIT = { "K", "M", "B", "T" };
+    private static final char[] NUM_UNIT = "kMGTPE".toCharArray();
```

Comment:
Why do you convert this field to char array? And convert these values to string later?
Please, restore original array and create new *string* array for your needs.